### DISCUSSION TITLE
✅ 🔗 Optimize work of off canvas widget with loop builder to create dynamic previews (Duplicate of #7106)

### DISCUSSION THREAD

**Discussion by Mesmer99** [elementor/elementor#28935] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

So on off canvas widget while experimenting with loop builder off canvas shows dynamic elements like post title or image correctly. Right now the problem is when you click to open of canvas it opens all loops, tho showing correct titles and images. This feature will be solution to problem that have history almost from beginning of elementor` create dynamic previews of posts or products without additional plugins or coding.

### Describe the solution you'd like.

The solution can be perfectly created based on off canvas widget. It needs just optimization explained above.

### Describe alternatives you've considered.

I think the popup solution is outdated based on off canvas widget capabilities and current optimal state.

### Additional context

Off canvas widget also needs some optimizations on functionality side like scroll disabling or multiple off canvas working together but in the end this is gamechanging feature that can be used with multiple purposes.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #7106

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.26.0! 🥳 

You can now use the **Off-Canvas** widget inside your **Loop Item** templates to display additional Dynamic Content for each post in your **Loop Grid** widgets! 🎉 🥳 🙌 

You can learn more about the Off-Canvas widget (and how to use it) here: 
https://elementor.com/help/off-canvas-widget/
And about the Loop Grid here: 
https://elementor.com/help/loop-grid/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
